### PR TITLE
Gpuciscripts clean and update

### PR DESCRIPTION
**Pull request by msadang**
In CI folder the following changes have been made:

- Remove logger function and replace all logger calls with gpuci_logger

- Removed all ellipses ... from logger messages

- Prepend /opt to the conda path in PATH variable

- Replace conda with gpuci_conda_retry for build and install calls  (Did not replace conda activate with gpuci_conda_retry) 

- Replace source activate <env> with the following:
. /opt/conda/etc/profile.d/conda.sh
conda activate rapids

- Replace conda list with more verbose information:
conda info
conda config --show-sources
conda list --show-channel-urls

- Update Copyright year in the top of scripts to include 2020 if applicable

- Set PARALLEL_LEVEL to ${PARALLEL_LEVEL:-4}

- Set gpuci_conda_retry flags in cpu/build.sh
Setup 'gpuci_conda_retry' for build retries (results in 2 total attempts)
export GPUCI_CONDA_RETRY_MAX=1
export GPUCI_CONDA_RETRY_SLEEP=30

- Replace calls to gcc and g++ with $CC and $CXX respectively

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]
It looks like my review comments from the `raft` PR below will apply here as well (remove `gpuci_logger` function and fix `export PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}`).

https://github.com/rapidsai/raft/pull/92#pullrequestreview-527348933

**Comment by dillon-cullinan** [comment]
Duplicate of: https://github.com/rapidsai/cuspatial/pull/314

**Comment by msadang** [comment]
> It looks like my review comments from the `raft` PR below will apply here as well (remove `gpuci_logger` function and fix `export PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}`).
> 
> [rapidsai/raft#92 (review)](https://github.com/rapidsai/raft/pull/92#pullrequestreview-527348933)

gpuci_logger and parallel level changed.

File: ci/cpu/build.sh [hunk 1]
```diff
@@ -5,19 +5,20 @@
...
 
-# Logger function for build status output
-function logger() {
-  echo -e "\n>>>> $@\n"
-}
-
 # Set path and build parallel level
-export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
-export PARALLEL_LEVEL=4
-export CUDF_HOME="${WORKSPACE}/cudf"
```
Comment (ajschmidt8) [inline ci/cpu/build.sh:16]: was `export CUDF_HOME="${WORKSPACE}/cudf"` intentionally removed here?

Comment (msadang) [reply ci/cpu/build.sh:16]: it wasn't intentional.  line was replaced.

**Comment by msadang** [comment]
rerun tests

**Pull request merged**